rust/kernel/lib.rs

```diff
@@ -12,7 +12,13 @@
 //! do so first instead of bypassing this crate.
 
 #![no_std]
-#![feature(allocator_api, alloc_error_handler, const_fn, const_mut_refs)]
+#![feature(
+    allocator_api,
+    alloc_error_handler,
+    const_fn,
+    const_mut_refs,
+    try_reserve
+)]
```

Comment:
Yeah, one per line is best for Git logs later. Even better, putting the full `#[feature(...)]` for each of them helps grepping for them + one less change when we remove all of them.